src/main/kotlin/blackjack/domain/betting/BettingResult.kt

```diff
@@ -0,0 +1,37 @@
+package blackjack.domain.betting
+
+import blackjack.domain.participant.Player
+import blackjack.domain.result.BlackjackResult
+import blackjack.domain.result.ResultType
+
+class BettingResult
+private constructor(private val bettingResult: Map<Player, Int>) {
```

Comment:
```suggestion
class BettingResult private constructor(private val bettingResult: Map<Player, Int>) {
```
클래스와 생성자는 한 줄에 같이 작성하는 것이 컨벤션이에요.
지금처럼 작성하시면, private 함수로 오해하기 쉬워요. 😅